services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy

```diff
@@ -204,21 +216,47 @@ class BandwidthSpec extends BaseSpecification {
                 .find { Switch src, Switch dst -> src != dst }
 
         when: "Create a flow with a bandwidth that exceeds available bandwidth on ISL (ignore_bandwidth = true)"
+        def linksBeforeFlowCreate = northboundService.getAllLinks()
         def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
-        long veryMaxBandwidth = northbound.getAllLinks()*.availableBandwidth.max()
-        flow.maximumBandwidth = veryMaxBandwidth + 1
+        long maxBandwidth = northbound.getAllLinks()*.availableBandwidth.max()
+        flow.maximumBandwidth = maxBandwidth + 1
         flow.ignoreBandwidth = true
         flowHelper.addFlow(flow)
         assert northboundService.getFlow(flow.id).maximumBandwidth == flow.maximumBandwidth
 
-        and: "Update the flow with a bandwidth that exceeds available bandwidth on ISL (ignore_bandwidth = true)"
-        flow.maximumBandwidth = veryMaxBandwidth + 2
+        then: "Available bandwidth on ISLs is not changed in accordance with flow maximum bandwidth"
+        def linksAfterFlowCreate = northboundService.getAllLinks()
+        def flowPath = PathHelper.convert(northboundService.getFlowPath(flow.id))
+
+        pathHelper.getInvolvedIsls(flowPath).each { link ->
+            [link, islUtils.reverseIsl(link)].each {
+                def bwBeforeFlowCreate = islUtils.getIslInfo(linksBeforeFlowCreate, it).get().availableBandwidth
+                def bwAfterFlowCreate = islUtils.getIslInfo(linksAfterFlowCreate, it).get().availableBandwidth
+                assert bwAfterFlowCreate == bwBeforeFlowCreate
+            }
+        }
+
+        when: "Update the flow with a bandwidth that exceeds available bandwidth on ISL (ignore_bandwidth = true)"
+        flow.maximumBandwidth = maxBandwidth + 2
         northboundService.updateFlow(flow.id, flow)
 
         then: "The flow is successfully updated and has 'Up' status"
         Wrappers.wait(WAIT_OFFSET) { assert northboundService.getFlowStatus(flow.id).status == FlowState.UP }
         northboundService.getFlow(flow.id).maximumBandwidth == flow.maximumBandwidth
 
+        and: "Available bandwidth on ISLs is not changed in accordance with new flow maximum bandwidth"
+        def linksAfterFlowUpdate = northboundService.getAllLinks()
+        def flowPathAfterUpdate = PathHelper.convert(northboundService.getFlowPath(flow.id))
+
+        flowPathAfterUpdate == flowPath
+        pathHelper.getInvolvedIsls(flowPathAfterUpdate).each { link ->
+            [link, islUtils.reverseIsl(link)].each {
+                def bwBeforeFlowCreate = islUtils.getIslInfo(linksBeforeFlowCreate, it).get().availableBandwidth
+                def bwAfterFlowUpdate = islUtils.getIslInfo(linksAfterFlowUpdate, it).get().availableBandwidth
+                assert bwAfterFlowUpdate == bwBeforeFlowCreate
+            }
```

Comment:
This pretty large code block repeats 5+ times in the spec. I think it will be nice to have it as a utility method on spec-level.

Reply:
done.